CondTools/HLT/plugins/AlCaRecoTriggerBitsRcdUpdate.cc

```diff
@@ -191,18 +211,109 @@ bool AlCaRecoTriggerBitsRcdUpdate::addTriggerLists(const std::vector<edm::Parame
   return true;
 }
 
+///////////////////////////////////////////////////////////////////////
+bool AlCaRecoTriggerBitsRcdUpdate::addPathsFromMap(const std::vector<edm::ParameterSet> &pathsToAdd,
+                                                   AlCaRecoTriggerBits &bits) const {
+  TriggerMap &triggerMap = bits.m_alcarecoToTrig;  //read from the condition tag
+
+  // loop on PSets, each containing the key (filter name) and a vstring with triggers
+  for (const auto &iSet : pathsToAdd) {
+    const std::string filter(iSet.getParameter<std::string>("listName"));
+    std::string mergedPathsInKey;
+
+    for (const auto &imap : triggerMap) {
+      if (imap.first == filter)
+        mergedPathsInKey = imap.second;  //paths in the condition tag
+    }
+
```

Comment:
As above, the check on `mergedPathsInKey.empty()` is missing.

Previously this was
```cpp
    if (mergedPathsInKey.empty()) {  // nothing to add ==> misconfiguration!
      throw cms::Exception("BadConfig") << "Didn't add any path to the map!";
    }
```

Reply:
This implementation was not proper for the intention. `mergedPathsInKey` is paths read from the condition tag, which is by design not empty. (at least ";")

Reply:
(check is done here):
https://github.com/cms-sw/cmssw/blob/a3d53a85e545e01a311bc28058fff692eba03067/CondTools/HLT/plugins/AlCaRecoTriggerBitsRcdUpdate.cc#L232-L234

Reply:
There seems to be some confusion: `mergedPathsInKey` is an empty `std::string` in L222; if `filter` does not match any of the keys in `triggerMap`, `mergedPathsInKey` will remain empty, which it looks like it should not happen.